spk/python/Makefile

```diff
@@ -1,36 +1,35 @@
 SPK_NAME = python
 SPK_SHORT_VERS = 2.7
-SPK_VERS =$(SPK_SHORT_VERS).13
-SPK_REV = 16
+SPK_VERS = $(SPK_SHORT_VERS).14
+SPK_REV = 17
 SPK_ICON = src/python.png
 
-DEPENDS  = cross/busybox cross/$(SPK_NAME)
+DEPENDS  = cross/$(SPK_NAME)
 # Build-time dependencies
 DEPENDS += cross/setuptools cross/pip cross/wheel
 DEPENDS += cross/cffi cross/cryptography cross/bcrypt
 
 # Cross-compiled wheels
 DEPENDS += cross/lxml cross/m2crypto cross/msgpack-python cross/pillow
-DEPENDS += cross/pocketsphinx cross/pyalsa cross/pyaudio cross/pycrypto cross/pycurl
-DEPENDS += cross/pyyaml cross/pyzmq cross/sphinxbase
-DEPENDS += cross/uwsgi cross/yenc
+DEPENDS += cross/pyalsa cross/pyaudio cross/pycrypto cross/pycurl
+DEPENDS += cross/pyyaml cross/pyzmq cross/uwsgi
 
 WHEELS = src/requirements.txt
 
-MAINTAINER = Diaoul
-DESCRIPTION = Python Programming Language.
-DESCRIPTION_FRE = Langage de programmation Python.
-DESCRIPTION_SPN = Lenguaje de programación Python.
+MAINTAINER = Safihre
```

Comment:
I appreciate you volonteer to maintain package. Seeing job you did on this package, I am confident about it.